dashboard/controller.py

```diff
@@ -88,6 +97,29 @@ def show_builds_history(page=1):
                            buildsets_log_url=buildset_log_url)
 
 
+@build_enqueue.route('/control')
+def control_panel():
+    if session and session['user'] and session['user'].is_admin():
+        return render_template('control_panel.html')
+    raise requests.codes.forbidden
```

Comment:
Don't use raise, error code is not an exception class. To raise HTTP error, use ``abort`` from flask. See: http://flask.pocoo.org/docs/0.12/api/#flask.abort